Cargo.toml

```diff
@@ -7,6 +7,8 @@ edition = "2021"
 
 [dependencies]
 bollard = "0.14"
+dotenv = "0.15.0"
 env_logger = "0.10.0"
 log = "0.4.19"
+postgres = "0.19.5"
```

Comment:
It has been discussed, but for the record the `postgres` crate is a *synchronous* client, thus not suitable for our needs.

Reply:
Done